framework/core/tests/integration/api/groups/ListTest.php

```diff
@@ -114,9 +114,9 @@ public function filters_only_public_groups_for_guest()
     {
         $response = $this->send(
             $this->request('GET', '/api/groups')
-            ->withQueryParams([
-                'filter' => ['hidden' => 0],
-            ])
+                ->withQueryParams([
+                    'filter' => ['hidden' => 0],
+                ])
```

Comment:
nitpick: just to keep the git history clean
```suggestion
            ->withQueryParams([
                'filter' => ['hidden' => 0],
            ])
```